tool-base/src/main/proto/spine/tools/java/java.proto

```diff
@@ -45,6 +45,9 @@ message JavaClassName {
 // A container for the Java classpath items.
 message Classpath {
 
-    // An absolute OS-dependent path to a particular classpath JAR.
-    repeated string jar = 1 [(required) = true];
+    // An OS-dependent path to a classpath item.
+    //
+    // A path could be relative or absolute.
+    //
+    repeated string item = 1 [(required) = true];
```

Comment:
@dmdashenkov This change looks good to me. 

Please have a look by yourself, too. Just to double-check, as I am not an often visitor in this part of our API.

Reply:
I've added `jars()` extension function to address the need of obtaining only JARs out of `items()` (which is another extension function just for the symmetry and being able to cross-reference in KDocs).